cordova-lib/src/plugman/fetch.js

```diff
@@ -144,7 +144,10 @@ function fetchPlugin(plugin_src, plugins_dir, options) {
         });
     }).then(function(result){
         checkID(options.expected_id, result.pinfo);
-        metadata.save_fetch_metadata(plugins_dir, result.pinfo.id, { source: result.fetchJsonSource });
+        var data = { source: result.fetchJsonSource };
+        data.is_top_level = options.is_top_level; 
```

Comment:
IMHO it's better to use an object literal here instead of assigning properties.

```javascript
var data = {
    source: result.fetchJsonSource,
    is_top_level: options.is_top_level,
    variables: options.variables || {}
};
```